ca_cdk_constructs/storage/aurora_fast_clone.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any
+from aws_cdk import CfnOutput, Token, Stack
+from constructs import Construct
+from aws_cdk.aws_ec2 import SubnetSelection, IVpc, SecurityGroup, IConnectable, Port
+from aws_cdk.aws_rds import (
+    CfnDBCluster,
+    DatabaseCluster,
+    SubnetGroup,
+    CfnDBParameterGroup,
+    CfnDBClusterParameterGroup,
+    CfnDBInstance,
+)
+
+
+class AuroraFastClone(Construct):
+    """Creates Aurora fast clone"""
+
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        source_cluster: DatabaseCluster,
+        vpc: IVpc,
+        db_instance_class: str,
+        cluster_parameters: dict[str, Any],
```

Comment:
make this optional (see below) and name `cluster_params` to be consistent with `instance_params`?

Reply:
can't, see above